src/components/Person/Person.jsx

```diff
@@ -1 +1,38 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    partnerName,
+    sex,
+    isMarried,
+  } = person;
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+        {age && (
+          <p className="Person__age">{`I am ${age}`}</p>
+        )
+        }
+
+        {isMarried ? (
+          <p className="Person__partner">
+            {`${partnerName} is my ${sex === 'm'
+              ? 'wife'
+              : 'husband'
+            }`}
+          </p>
+        ) : (
+          <p className="Person__partner">
+            I am not married
+          </p>
+        )
+        }
```

Comment:
Avoid using nested conditions. Create variable for `${sex === 'm' ? 'wife' : 'husband'}`